src/emu/epoc/src/services/fbs/adapter/stb_font_adapter.cpp

```diff
@@ -168,7 +173,7 @@ namespace eka2l1::epoc::adapter {
             return nullptr;
         }
 
-        std::uint8_t *result = nullptr;
+        std::uint8_t *result;
```

Comment:
Why not? This just leave garbage pointer

Reply:
`result` will be initialized right after that so I thought nullptr there is kinda redundant as it would be never used.

Reply:
Fair enough, if it does not throw compiler warning im fine